source/Playnite.DesktopApp/ViewModels/SettingsViewModel.cs

```diff
@@ -349,6 +350,23 @@ public RelayCommand ResetDateTimeFormatLastPlayedCommand
             });
         }
 
+        public RelayCommand UploadFullscreenIntroVideoCommand
+        {
+            get => new RelayCommand(() =>
+            {
+                string filePath = Dialogs.SelectFile("MP4 files (*.mp4)|*.mp4|All files (*.*)|*.*");

```

Comment:
"MP4 files (*.mp4)" and "All files (*.*)" are technically user-facing strings and arguably could be translatable. Let me know if that is something worth separating out. I've done it in the past, so it's no big deal.